app/Core/Database.php

```diff
@@ -1,5 +1,6 @@
 <?php
 namespace Core;
+$config_data = parse_ini_file("dbconfig.ini");
```

Comment:
This should be in the `initialize()` method

Reply:
done